Sources/NIOSSH/User Authentication/UserAuthenticationStateMachine.swift

```diff
@@ -186,6 +186,10 @@ extension UserAuthenticationStateMachine {
             throw NIOSSHError.protocolViolation(protocolName: Self.protocolName, violation: "client sent user auth failure")
         }
     }
+
+    mutating func receiveUserAuthBanner(_ message: SSHMessage.UserAuthBannerMessage) throws {
+      // TODO: implement validation
```

Comment:
Was this missed, or did you want to discuss it?

Reply:
I indeed must have forgotten to implement this functionality. Thanks for the heads up